packages/web/integrations/nomic/transfer.tsx

```diff
@@ -33,6 +33,13 @@ export const displayBtc = (num: number): string => {
   return resStr.replace(/\.?0+$/, "") + " BTC";
 };
 
+const calculateMinDeposit = (
+  minerFeeSats: number | undefined,
+  bridgeFee: number | undefined
+): number => {
+  return 1000 / (1 - (bridgeFee as number)) + (minerFeeSats as number) * 1e8;
+};
```

Comment:
We need to manage undefined values effectively. Perhaps we can default both bridgeFee and minerFeeStas to 0 or simply exit early?

Reply:
I didn't want an improper value to stick in the case of race condition. Defaulting to 0 will always yield a result that is less than the actual minimum deposit. In any case, since `minerFee` and `bridgeFee` are pulled from the same response as `reachedCapacityLimit`, we can be sure that the user won't be able to proceed to the deposit page without the client first setting those values. However, I've updated this to make that contract more explicit.